engines/download_only.py

```diff
@@ -30,6 +30,7 @@ class engine(Engine):
                      ]
 
     def table_exists(self, dbname, tablename):
+        """Checks if the given table exists"""
```

Comment:
"""Checks if the given table exists""" -> """Checks if the file to be downloaded already exists"""